os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -141,7 +141,8 @@ void board_spi_initialize(void)
 	}
 #endif
 #endif
-#ifdef CONFIG_AMEBASMART_SPI1
+
+#if defined(CONFIG_AMEBASMART_SPI1) &&  !defined(CONFIG_SECOND_FLASH_PARTITION)
```

Comment:
How can we make sure that second flash will be connected to SPI1 always?

Reply:
if another external flash attach to another spi, then code need to be expanded in future, I just aligned code with current setting.